co-simulation/fed/mosaic-infrastructure/src/main/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureInstanceManager.java

```diff
@@ -91,25 +101,30 @@ public void onNewRegistration(InfrastructureRegistrationMessage registration) {
      * 
      */
     private void newInfrastructureInstance(String infrastructureId, InetAddress rxMessageIpAddress, int rxMessagePort,
-            int timeSyncPort, CartesianPoint location) {
+            int timeSyncPort, int simulatedInteractionPort, CartesianPoint location, List<Detector> sensors) {
         InfrastructureInstance tmp = new InfrastructureInstance(infrastructureId, rxMessageIpAddress, rxMessagePort,
-                timeSyncPort, location);
+                timeSyncPort, simulatedInteractionPort, location, sensors);
         try {
             tmp.bind();
-            log.info("New Infrastructure instance '{}' registered with Infrastructure Instance Manager.", infrastructureId);
+            log.info("New Infrastructure instance '{}' registered with Infrastructure Instance Manager.",
+                    infrastructureId);
         } catch (IOException e) {
             log.error("Failed to bind infrastructure instance with ID '{}' to its RX message socket: {}",
                     infrastructureId, e.getMessage());
             log.error("Stack trace:", e);
         }
+
         managedInstances.put(infrastructureId, tmp);
     }
 
     /**
-     * Callback to be invoked when CARMA Platform receives a V2X Message from the NS-3 simulation
+     * Callback to be invoked when CARMA Platform receives a V2X Message from the
+     * NS-3 simulation
+     * 
      * @param sourceAddr The V2X Message received
-     * @param txMsg The Host ID of the vehicle receiving the data
-     * @throws RuntimeException If the socket used to communicate with the platform experiences failure
+     * @param txMsg      The Host ID of the vehicle receiving the data
+     * @throws RuntimeException If the socket used to communicate with the platform
+     *                          experiences failure
      */
     public V2xMessageTransmission onV2XMessageTx(InetAddress sourceAddr, CarmaV2xMessage txMsg, long time) {
```

Comment:
"onV2xMessageTx" What does the Tx stands for?

Reply:
There is a time parameter. What is unit of measure for this time? What is purpose of this time in this function?

Reply:
Rx and Tx universally mean transmit and receive. The time refers to simulation time, specifically the time for the transmission interaction. This method is also not part of my PR. I think this review should focus on code changes.